lib/libtlspool.h

```diff
@@ -0,0 +1,19 @@
+/* The request registry is an array of pointers, filled by the starttls_xxx()
```

Comment:
ahhh, je exporteert vanalles van een interne API hiermee, niet akkoord!

Reply:
wat je zou kunnen doen is de functionlaliteit per OS met #include inladen -- dat werkt ook op C files

Reply:
dat doe ik nu, mooie oplossing!